3Dmol/glmodel.js

```diff
@@ -2444,6 +2444,34 @@ $3Dmol.GLModel = (function() {
             }
         }
 
+
+    /**
+    * Set coordinates for the atoms parsed from various topology files. 
+    * @function $3Dmol.GLModel#setCoordinatesFromUrl
+    * @param {string} url - contains the url where mdsrv has been hosted
+    * @param {string} path - contains the path of the file (<root>/filename)
+    * @param {object} viewer - contains the glviewer object
+    */
+
+        this.setCoordinatesFromUrl = function(url, path, viewer) {
```

Comment:
setCoordinates should not animate the model.  It definitely shouldn't set the style. The user should be able to call animate using the standard api.  It should setup data structures appropriately so that the coordinate data can be fetched _as needed_ when the frame position is changed in `setFrame`.  There is no need to pass the viewer.